docs/admin/src/widgets/variationPreviewTemplate.js

```diff
@@ -12,31 +15,57 @@ const templateWithIcons = template.replace(
   ( match, icon ) => require( `../../../_includes/icons/${ icon }.svg` )
 );
 
+/**
+ * @param {MouseEvent} event - The mouse event object from the click.
```

Comment:
This orphan jsdoc looks like it was associated with a function that was deleted or moved.